### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by fippli**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (Jim-Holmstroem) [inline LICENSE:189]: Fill out :)

Comment (fippli) [inline LICENSE:189]:
```suggestion
   Copyright 2020 Aiwizo
```

**Review by Jim-Holmstroem — APPROVED** [review]

**Review by Jim-Holmstroem — APPROVED** [review]

**Comment by Jim-Holmstroem** [comment]
does npm pick up the LICENSE file automatically or do you have to register somewhere to have it added?

**Comment by fippli** [comment]
I add it in package.json

**Comment by Jim-Holmstroem** [comment]
Sure its marked as Apache 2, but is the LICENSE file actually distributed, I know in python you have to actually add the file as well

**Pull request merged**